### PR TITLE
Add syntax highlighting for python in documentation

### PR DESCRIPTION
**Pull request by Silmathoron**
This PR adds correct highlighting for python code blocks.

### CODE REVIEW

**Review by jougs — APPROVED** [review]

**Comment by clinssen** [comment]
I still have an open PR against @Silmathoron's branch, please don't merge just yet!

**Comment by Silmathoron** [comment]
Ah, sorry, did not see it was a PR on my branch ^^"
Just rebased it

**Comment by jougs** [comment]
No worries, I just wanted to indicate my approval. Please merge yourself whenever you think it's ready.

File: doc/tutorial/izhikevich_tutorial.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Start your Python interpreter
 
 Note that on MacOS, `LD_LIBRARY_PATH` is called `DYLD_LIBRARY_PATH`.
 
-.. code::
+.. code-block:: python
```
Comment (clinssen) [inline doc/tutorial/izhikevich_tutorial.rst:14]:
```suggestion
.. code-block:: bash
```
My bad.

**Review by clinssen — CHANGES_REQUESTED** [review]

**Comment by clinssen** [comment]
Many thanks!

**Pull request merged**